frontend/datasafe-ui/src/app/service/api/api.service.ts

```diff
@@ -41,6 +41,7 @@ export class ApiService {
     }
 
     createUser(username: string, password: string) {
+        console.log("send message to " + this.createUserUri);
```

Comment:
Is it needed?

Reply:
like logs are never needed. When angular run stand alone I was not able to reach the server. And the app did not tell me who it was calling.

Reply:
The way it is written for me looked like lost log statement.
I suggest to rewrite this to:
```
console.info(`Creating user using api URL '${this.uri}'`);
```
It is very misleading right now and ignores log level

Reply:
did not know, that console has separate log levels. thanks